data/korath/korath ships.txt

```diff
@@ -851,6 +851,213 @@ ship "Korath World-Ship" "Korath World-Ship C"
 	turret 91 290 "Korath Warder"
 
 
+ship "Korath Haven"
+	sprite "ship/korath haven"
+	"uncapturable"
+	attributes
+		category "Heavy Freighter"
+		cost 27690000
+		shields 196420
+		hull 320690
+		"required crew" 1940
+		"bunks" 10920
```

Comment:
If it's uncapturable we don't need to worry about the calculation, right?  This ought to be a few times larger.

```suggestion
		"bunks" 31920
```